### PR TITLE
add kms performance parameters

### PR DESCRIPTION
**Pull request by ramonpetgrave64**
Editing the `KMSTokenValidator` to allow the boto `Config` object to customizable.

For a web server use case, such as within [lyft/confidant](https://github.com/lyft/confidant/blob/59bae423ae9c7c3f8a2967548e2c9d812c97729a/confidant/settings.py#L460-L468), some good values are

```
max_pool_connections=100,
connect_timeout=1,
read_timeout=1,
```

The [boto defaults](https://botocore.amazonaws.com/v1/documentation/api/latest/reference/config.html) are 10, 60, 60 respectively.

### CODE REVIEW

File: kmsauth/__init__.py [hunk 1]
```diff
@@ -57,6 +57,9 @@ def __init__(
             endpoint_url=None,
             token_cache_size=4096,
             stats=None,
+            max_pool_connections=100,
```
Comment (skiptomyliu) [inline kmsauth/__init__.py]: we should make these defaults to match the boto defaults.  Then override as needed.

Comment (alejandroroiz) [reply kmsauth/__init__.py]: https://botocore.amazonaws.com/v1/documentation/api/latest/reference/config.html

defaults
`max_pool_connections` = 10
`connect_timeout ` = 60 (seconds)
`read_timeout ` = 60 (seconds)

(whoops just saw you already had this in the description, mb)

File: kmsauth/__init__.py [hunk 2]
```diff
@@ -57,6 +57,9 @@ def __init__(
...
+            connect_timeout=1,
+            read_timeout=1,
             ):
         """Create a KMSTokenValidator object.
```
Comment (alejandroroiz) [inline kmsauth/__init__.py:65]: we should probably also document these new parameters

**Review by skiptomyliu — DISMISSED** [review]

File: kmsauth/__init__.py (hunk 2 above)
Comment (ramonpetgrave64) [reply kmsauth/__init__.py:65]: added

**Review by alejandroroiz — APPROVED** [review]

**Pull request merged**